storybook/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -179,28 +182,40 @@ export const CheckboxGroup: Story = {
   ),
 }
 
-// export const CheckboxGroupWithError: Story = {
-//   args: {
-//     invalid: true,
-//     legend: 'Waar gaat uw melding over?',
-//   },
-//   render: (args) => (
-//     <FieldSet legend={args.legend} invalid={args.invalid}>
-//       {args.invalid && <ErrorMessage className="ams-mb--sm">Geef aan waar uw melding over gaat.</ErrorMessage>}
-//       <Column gap="extra-small">
-//         <Checkbox name="about" value="horeca" invalid={args.invalid} aria-required="true">
-//           Horecabedrijf
-//         </Checkbox>
-//         <Checkbox name="about" value="ander_bedrijf" invalid={args.invalid} aria-required="true">
-//           Ander soort bedrijf
-//         </Checkbox>
-//         <Checkbox name="about" value="evenement" invalid={args.invalid} aria-required="true">
-//           Evenement
-//         </Checkbox>
-//         <Checkbox name="about" value="anders" invalid={args.invalid} aria-required="true">
-//           Iets anders
-//         </Checkbox>
-//       </Column>
-//     </FieldSet>
-//   ),
-// }
+export const CheckboxGroupWithError: Story = {
+  args: {
+    invalid: true,
+    legend: 'Waar gaat uw melding over?',
+  },
+  render: (args) => (
+    <FieldSet
+      aria-labelledby="fieldset2 description7 error4"
```

Comment:
Here the same comment as earlier.

I'm not sure about updating the label with the error message if it appears. Why would you want to do that?

Reply:
Same answer: so it's read by screen readers